server/pulp/server/db/model/dispatch.py

```diff
@@ -489,3 +490,27 @@ def __init__(self, call_request, call_report):
         self.timestamp = dateutils.now_utc_timestamp()
         self.call_request_string = str(call_request)
         self.serialized_call_report = call_report.serialize()
+
+
+class CeleryBeatLock(Document):
+    """
+    Single document collection which gives information about the current celerybeat lock.
+
+        :param celerybeat_name: string representing the celerybeat instance name
+        :type  celerybeat_name: basestring
+        :param timestamp: The timestamp(UTC) at which lock is acquired
+        :type  timestamp: datetime.datetime
+        :param lock: A unique key set to "locked" when lock is acquired.
+        :type  lock: basestring
+    """
+    celerybeat_name = StringField(required=True)
+    timestamp = DateTimeField(required=True)
+    lock = StringField(required=True, default="locked")
```

Comment:
You should add a _ns option here for backwards compatibility. Add these two lines with a space above and below:

```
    # For backward compatibility
    _ns = StringField(default='celery_beat_lock')
```